src/author-tags.js

```diff
@@ -8,7 +8,7 @@ import {get} from 'lodash';
  * @param {...*} params See {@link Generator} for other Parameters
  */
 export function AuthorTags(seoConfig, config, pageType, data, {url}) {
-  if(pageType != 'story-page')
+  if(pageType != 'story-page' || pageType != 'story-page-amp')
```

Comment:
This is not a scalable model. Better make the pageType as `amp-page`.

Reply:
named it `story-page-amp` because thought we might later want to add more pageTypes for amp section, home pages etc.